packages/x-tailwindcss/src/x-tailwind-plugin/components/badge/colors.ts

```diff
@@ -0,0 +1,24 @@
+import { TailwindHelpers } from '../../../types';
+import { mapColors } from '../../utils/map-colors';
+
+/**
+ * Returns the `colors` variants for component `badge`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function badgeColors(helpers: TailwindHelpers) {
+  const { theme } = helpers;
+
+  return {
+    ...mapColors(color => {
+      return {
+        '--badge-color-25': color['25'],
+        '--badge-color-50': color['50'],
+
+        backgroundColor: color['75'],
+        color: theme('colors.neutral.0')
```

Comment:
is this needed? because the default one is applying this style too

Reply:
Yep, it is not needed